zerver/models.py

```diff
@@ -3336,6 +3336,7 @@ class AbstractRealmAuditLog(models.Model):
     REALM_SPONSORSHIP_PENDING_STATUS_CHANGED = 213
     REALM_SUBDOMAIN_CHANGED = 214
     REALM_CREATED = 215
+    REALM_IMPORTED = 216
```

Comment:
Do we have a theory for how we could identify which realms were imported in the past for a backfilled migration on that front?

I guess the best idea I have on that front is that we could look at the plan_type records for when organizations changed from `SELF_HOSTED`.

Also, it'd be really nice if these records recorded what a realm was imported from (E.g. was it Slack or another Zulip server); so maybe we should adjust the export format to include that detail (if it doesn't already), and then include that in `extra_data` for this PR?